sci-physics/collier/metadata.xml

```diff
@@ -17,6 +17,6 @@
   the 2-, 3-, and 4-point tensor coefficient functions.
   </longdescription>
   <use>
-    <flag name="static">Build static library</flag>
+    <flag name="static-libs">Build static library</flag>
```

Comment:
This flag is already described globally (in `::gentoo`), so there is no need to describe it locally here.

Reply:
That explains why `pkgcheck` did not complain before.